dsc/authoringAdvanced.md

```diff
@@ -0,0 +1,171 @@
+# Advanced DSC Authoring for Composition and Collaboration
+
+This article describes the types of approaches available
+for combining configurations and resources.
+The goal for each scenario is the same,
+to reduce complexity when multiple configurations
+are preferred to reach a server deployment end state.
+An example of this would be multiple teams contributing
+to the outcome of a server deployment,
+such as an application owner maintaining the application state
+and a central team releasing changes to security baselines.
+The nuances of each approach including the benefits and risks
+are detailed here.
+
+![Pipeline](images/Pipeline.jpg)
+
+## Types of Collaborative Authoring Techniques
+
+There are two solutions built in to Local Configuration Manager
```

Comment:
Terminology is very confusing. Solutions, approaches, concepts... Do they refer to the same thing or not? I'm lost.